superset/assets/cypress_build.sh

```diff
@@ -12,5 +12,5 @@ cd "$(dirname "$0")"
 npm install -g yarn
 yarn
 npm run build
-npm run cypress run --record --parallel --key 1f958c86-be14-44d9-8d08-fad68da06811
+npm run cypress run --record
```

Comment:
I was looking at ci logs and realize that none of the flags are passed to cypress.
The `--record` `--parallel` and `--key` flags were/are passed to npm. 
To pass them to cypress, have to add `--` first.

```
npm run cypress run -- --record
```